global.go

```diff
@@ -18,30 +18,36 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
+// Debug logs a message at DebugLevel. The message includes any fields passed
+// at the log site, as well as any fields accumulated on the logger.
+func Debug(msg string, fields ...zap.Field) {
+	L().WithOptions(zap.AddCallerSkip(1)).Debug(msg, fields...)
```

Comment:
Could we add this option to the logger directly when creating the logger?

Reply:
Yes, we can. But I do not directly create the logger with this option because the user may use `L()`,`S()` without the export function.